pkg/apis/stork/v1alpha1/applicationbackup.go

```diff
@@ -93,6 +93,12 @@ type ApplicationBackupResourceInfo struct {
 	ObjectInfo `json:",inline"`
 }
 
+// This structure stores metadata related to PSA feature that is required for volume backup/restore
```

Comment:
I would have added more specifically who is the consumer 
// This object is used in VolumeInfo for PSA enabled cluster to retain the runAsUser ID and runAsGroup ID used by
//  Job pod(KDMP/NFS)during backup. We will use the same IDs to spin up Job Pods during restore.

Reply:
Done as suggested